test/README.md

```diff
@@ -16,34 +43,36 @@ make
 # or
 make edge_core
 ```
+### Modify the configuration files accordingly
 
-### Install dependency
-
-You need install **mosquitto** to support mqtt.
-
-xref: https://mosquitto.org/download/
-
-```shell
-# For ubuntu
-apt install -y mosquitto
-```
+#### Configure
 
-### Configure
+##### `modules.yaml` (add the `testManager`)
 
-#### Modify the configuration files accordingly
+##### Reason to add `testManager` in modules.yaml:
 
-##### `modules.yaml` (add the `testManager`)
+Kubeedge uses **beehive framework** as the module to module communication, all modules in the kubeedge has to register with beehive for internal communication.
```

Comment:
KubeEdge uses beehive framework for inter-module communicaton. All modules need to register with beehive .
testManager is a module like other KubeEdge modules. So, it has to be configured as shown below :